### PR TITLE
Reduce logging from network layer

### PR DESCRIPTION
**Pull request by rvl**
### Issue Number

Relates to #1577 / ADP-83.

### Overview

- Set network tip log level to debug
- Improve formatting of text log message
- Only log the tip when it changes, because the node can seem to send updates when nothing changes

### Comments

This is what it looks like now:

```
[cardano-wallet.network:Debug:18] [2020-05-06 07:49:54.10 UTC] Network node tip block height is 4125310 at hash c5cb3bcd
```

### CODE REVIEW

File: lib/byron/src/Cardano/Wallet/Byron/Network.hs [hunk 1]
```diff
@@ -497,6 +494,11 @@ mkTipSyncClient tr localTxSubmissionQ onTipUpdate onTxParamsUpdate = do
             Right (Right ls) ->
                 onTxParamsUpdate' $ txParametersFromUpdateState ls
 
+    onTipUpdate' <- debounce $ \tip -> do
```
Comment (jonathanknowles) [inline lib/byron/src/Cardano/Wallet/Byron/Network.hs:497]: :+1:

**Review by jonathanknowles — APPROVED** [review]
Nice!

I particularly like the use of the `debounce` function.

**Comment by rvl** [comment]
Thanks

bors r+

**Pull request merged**